Integrations/HelloWorld/HelloWorld.py

```diff
@@ -0,0 +1,216 @@
+        """
+        initiates a http request to test url
+        """
+        data = self._http_request(
+            method='GET',
+            url_suffix='/hello/' + name
+        )
+        return data.get('result')
+
+    def list_incidents(self):
+        """
+        returns dummy incident data, just for the example.
+        """
+        return [
+            {
+                'incident_id': 1,
+                'description': 'Hello incident 1',
+                'created_time': datetime.utcnow().strftime(DATE_FORMAT)
+            },
+            {
+                'incident_id': 2,
+                'description': 'Hello incident 2',
+                'created_time': datetime.utcnow().strftime(DATE_FORMAT)
+            }
+        ]
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test
+    """
+
+    result = client.say_hello('DBot')
+    if 'Hello DBot' == result:
+        return 'ok'
+    else:
+        return 'Test failed because ......'
+
+
+def say_hello_command(client, args):
+    """
+    Returns Hello {somename}
+
+    Args:
+        client: HelloWorld client
+        args: all command arguments
+
+    Returns:
+        Hello {someone}
+
+        readable_output: This will be presented in Warroom - should be in markdown syntax - human readable
+        outputs: Dictionary/JSON - saved in incident context in order to be used as input for other tasks in the
+                 playbook
+        raw_response: Used for debugging/troubleshooting purposes - will be shown only if the command executed with
+                      raw-response=true
+    """
+    name = args.get('name')
+
+    result = client.say_hello(name)
+
+    # readable output will be in markdown format - https://www.markdownguide.org/basic-syntax/
+    readable_output = f'## {result}'
+    outputs = {
+        'hello': result
+    }
+
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def say_hello_over_http_command(client, args):
+    name = args.get('name')
+
+    result = client.say_hello_http_request(name)
+
+    # readable output will be in markdown format - https://www.markdownguide.org/basic-syntax/
+    readable_output = f'## {result}'
+    outputs = {
+        'hello': result
+    }
+
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def fetch_incidents(client, last_run, first_fetch_time):
+    """
+    This function will execute each interval (default is 1 minute).
+
+    Args:
+        client: HelloWorld client
+        last_run: The greatest incident created_time we fetched from last fetch
+        first_fetch_time: If last_run is None then fetch all incidents since first_fetch_time
```

Comment:
```suggestion
        client (Client): HelloWorld client
        last_run (dateparser.time): The greatest incident created_time we fetched from last fetch
        first_fetch_time (dateparser.time): If last_run is None then fetch all incidents since first_fetch_time
```